testcases/ltp-syscalls-kasan.yaml

```diff
@@ -2,4 +2,4 @@
 
 {% set testnames = ['syscalls'] %}
 {% set test_timeout = 120 %}
-{% set ltp_name_suffix = '-kasan' %}
+{% set LAVA_TEST_NAME_SUFFIX = '-kasan' %}
```

Comment:
I guess this file can be removed now. But it requires changes to test plans as well.

Reply:
This file will be removed when I've changed our ci system not to calll for ltp-syscalls-kasan.yaml
but I can only do that when the variable are in place.